src/pobox.erl

```diff
@@ -337,3 +488,26 @@ pop(stack, []) -> {empty, []};
 pop(stack, [H|T]) -> {{value,H}, T};
 pop(keep_old, Q) -> queue:out(Q);
 pop({mod, Mod}, Data) -> Mod:pop(Data).
+
+
+send_ownership_transfer(_PreviousOwner, none, _HeirData, _BoxName) -> {error, no_process};
+send_ownership_transfer(_PreviousOwner, undefined, _HeirData, _BoxName) -> {error, no_process};
+send_ownership_transfer(PreviousOwner, NewOwnerName, HeirData, BoxName)
+    when ?PROCESS_NAME_GUARD_WITH_LOCAL_NO_PID(NewOwnerName), ?PROCESS_NAME_GUARD(BoxName) ->
+    case where(NewOwnerName) of
+        Pid when is_pid(Pid) ->
+            Pid ! {pobox_transfer, BoxName, PreviousOwner, HeirData},
```

Comment:
Might be unsafe to provide only `BoxName` instead of `pid()`. Should we also provide some sort of information about what triggered the transfer. For example crash vs normal exit vs give_away?

Reply:
fixed